client-v2/src/bundles/capiFeedBundle.ts

```diff
@@ -113,11 +115,58 @@ export const fetchPreview = (
   }
 };
 
+const {
+  actions: prefillActions,
+  reducer: prefillFeed,
+  selectors: prefillSelectors
+} = createAsyncResourceBundle<FeedState>('prefillFeed', {
+  indexById: false,
+  selectLocalState: state => state.feed.prefillFeed,
+  initialData: []
+});
+
+export const fetchPrefill = (
+  id: string
+): ThunkResult<void> => async dispatch => {
+  dispatch({
```

Comment:
We should make this an action creator! Then we can derive the type using ReturnType<typeof actionCreator> and reduce the number of duplicated strings (see e.g. types/Action)